src/src/Helpers/StringParser.cpp

```diff
@@ -26,6 +26,20 @@
 /********************************************************************************************\
    Parse string template
  \*********************************************************************************************/
+// Checks str for the given escaped characters
+// So far \\% \\[ and \\] are used (all with single backslash!)
+bool hasEscapeCharacters(String *str, const String EscapeChar)
+{
+return (str->indexOf(EscapeChar)>=0);
+}

```

Comment:
That `const String` can be a `const __FlashStringHelper`, that will require those arguments to have the `F()` macro applied.

And the `return` can be indented (please don't Uncrustify this file, that'll cause hundreds of spacing changes...)